zebra-chain/src/block/tests/preallocate.rs

```diff
@@ -0,0 +1,97 @@
+//! Tests for trusted preallocation during deserialization.
+
+use super::super::{
+    header::MIN_COUNTED_HEADER_LEN, CountedHeader, Hash, Header, BLOCK_HASH_SIZE,
+    MAX_PROTOCOL_MESSAGE_LEN,
+};
+
+use crate::serialization::{TrustedPreallocate, ZcashSerialize};
```

Comment:
Smush these together?

```suggestion
use crate::{
   block::{
       header::MIN_COUNTED_HEADER_LEN, CountedHeader, Hash, Header, BLOCK_HASH_SIZE,
       MAX_PROTOCOL_MESSAGE_LEN,
    },
    serialization::{TrustedPreallocate, ZcashSerialize}
};
```